app/components/Views/confirmations/ApproveView/Approve/index.js

```diff
@@ -852,6 +858,10 @@ class Approve extends PureComponent {
             address={address}
             savedContactListToArray={savedContactListToArray}
             addressNickname={addressNickname}
+            providerType={providerType}
+            providerChainId={providerChainId}
```

Comment:
I think this should be the transaction chainId, that would be the right usage, can you confirm @OGPoyraz ?
```suggestion
            providerChainId={chainId}
```

Reply:
Yes I think it should be `chainId` that comes from meta